allauth/socialaccount/models.py

```diff
@@ -97,17 +157,20 @@ class SocialAccount(models.Model):
     # [2] http://openid.net/specs/openid-authentication-1_1.html#limits
 
     uid = models.CharField(verbose_name=_('uid'),
-                           max_length=app_settings.UID_MAX_LENGTH)
+                           max_length=app_settings.UID_MAX_LENGTH,
+                           db_index=True)
```

Comment:
Adding an index is beyond the scope of this PR. Why is it needed, as there is a unique together on provider, uid ?